lib/utils.js

```diff
@@ -47,14 +47,15 @@ export function storeToTtl(store) {
  * is not big. Some datatypes are less explicit in Turtle, such as booleans,
  * which causes mu-authorization to fail.
  */
-function storeToString(store, forSparql = true) {
+async function storeToString(store, forSparql = true) {
```

Comment:
since this function returns a promise I thought it was more clear to declare it async, though not strictly required